dart/test/mocks/mock_transport.dart

```diff
@@ -28,35 +43,13 @@ class MockTransport implements Transport {
 
     final envelopeItem = utf8.decode(envelopeItemData);
     final envelopeItemJson = jsonDecode(envelopeItem.split('\n').last);
-    final envelopeMap = envelopeItemJson as Map<String, dynamic>;
-    final requestJson = envelopeMap['request'] as Map<String, dynamic>?;
-
-    // '_InternalLinkedHashMap<dynamic, dynamic>' is not a subtype of type 'Map<String, String>'
```

Comment:
We have the same issue on dio/test/mocks/mock_transport as well, Want to remove it from there as well? since the ctor is safe now.

Reply:
OK, synced the mock_transport.dart copy in DIO